content/influxdb/v1.3/write_protocols/line_protocol_reference.md

```diff
@@ -6,6 +6,7 @@ menu:
   influxdb_1_3:
     weight: 10
     parent: Write protocols
+canonical: /{{< latest "influxdb" "v2" >}}/reference/syntax/line-protocol/
```

Comment:
Scott, did you mean to add this only to line protocol reference docs?

Reply:
Yes, just for now. We will need to identify other pages that will benefit from this. This was the page I thought of first and used to test. We certainly will want to add this frontmatter to other pages.